pkg/wasm/example/wasm_exec.js

```diff
@@ -0,0 +1,554 @@
+// Copyright 2018 The Go Authors. All rights reserved.
```

Comment:
Can we document where this was sourced from or maybe even have the example load it from there?

Reply:
The HTML file actually references its original location. The problem with loading the JS from GitHub is the requirement to, well, load it from GitHub, which is non-ideal, especially if it changes

Reply:
If you press `y` on a GitHub page it gives you the absolute URL -- that would prevent from any changes outside of force pushes or repository deletions.

This is just an example, so I'm not super concerned.

Reply:
Good to know